### ISSUE TITLE
Dashboard Improvements

### ISSUE THREAD

**Issue by dsibiski** [AutoElevate/roadmap#15]

**Comment by ocnjkayak** [comment]
Yes absolutely would love to be able to click-through the dashboard to review the underlying data!

**Comment by scubes13** [comment]
I'd also add that it would be great to add a dashboard graph that takes into account the use of "Policy" mode. Right now, this is the mode that we default to. We do not use the "Active" mode. So, it appears at first glance as if we're not doing anything - until you inspect further.
